.rubocop.yml

```diff
@@ -4,6 +4,7 @@ require:
 AllCops:
   DisplayCopNames: true
   DisplayStyleGuide: true
+  NewCops: enable
```

Comment:
This automatically opts us into new rules that are added in future versions of rubocop. So we no longer need to explicitly list the new cops with `Enabled: true` every time we update the gems.

Reply:
❤️